spec/requests/eligibilities_spec.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require 'rails_helper'
+require 'swagger_helper'
```

Comment:
I think you can just put `require 'swagger_helper`` inside of either `rails_helper.rb` or `spec_helper.rb` and it'll automatically get required. That way, you won't have to put `require 'swagger_helper'` in all of these spec files. Or alternatively, you could just replace the `require 'rails_helper'` calls because `swagger_helper.rb` already requires `rails_helper`.